metricbeat/module/windows/perfmon/reader.go

```diff
@@ -124,6 +129,44 @@ func (re *Reader) Read() ([]mb.Event, error) {
 	return events, nil
 }
 
+func (re *Reader) getValues() (map[string][]pdh.CounterValue, error) {
+	var interval uint32 = 10
+	var val map[string][]pdh.CounterValue
+	rand.Seed(time.Now().UnixNano())
+	title := windows.StringToUTF16Ptr(randSeq(10))
+	event, err := windows.CreateEvent(nil, 0, 0, title)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
A comment here would be helpful. Is this `interval` in seconds? Should this be related to the metricset period?

Reply:
should be the metricset period, forgot to map it, all good now